accepted/PSR-7-http-message.md

```diff
@@ -1606,8 +1606,9 @@ interface UriInterface
      *
      * @param string $scheme The scheme to use with the new instance.
      * @return self A new instance with the specified scheme.
-     * @throws \InvalidArgumentException for invalid schemes.
```

Comment:
This is covered by the more generic wording I added. So no need for this anymore.